interview-questions.md

```diff
@@ -7,53 +7,170 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a hash in Ruby?
 
-  Your answer:
+  Your answer: A hash is a key in ruby. It can be used to tag a collection of data.
 
-  Researched answer:
+  Researched answer: 
```

Comment:
Loving this researched answer!!!  Just make sure that your answer that you would provide in an interview displays your thorough knowledge as well, perhaps by providing some context on when you have used hashes.